autogpt/utils.py

```diff
@@ -11,10 +11,55 @@
 except:
     pass
 
+from autogpt.config import Config
 
-def clean_input(prompt: str = ""):
+cfg = Config()
+
+
+def send_chat_message_to_user(report: str):
+    if cfg.chat_messages_enabled:
+        for plugin in enumerate(cfg.plugins):
```

Comment:
A less indentation-heavy version of this would be:
```suggestion
    if not cfg.chat_messages_enabled:
        return

    for plugin in enumerate(cfg.plugins):
```

Just a point of feedback, doesn't necessarily need fixing.

Reply:
accepted, I agree

Reply:
Well in that case, https://github.com/Significant-Gravitas/Auto-GPT/pull/2929/files#diff-680674214fd6efa66f980e3ba510374dc0adb02690fd4e642c637bcf669332ffR32-R33 has the same structure :p